### PR TITLE
Bookstore Project New Milestone : Add redux to react

### PR DESCRIPTION
**Pull request by toyman640**
- I added the previously created redux reducer to my react project and the initial state displays the list of books.

### CODE REVIEW

File: src/components/Addbook.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+          payload: {
+            id: '',
+            title: '',
+            author: '',
+          },
+        })}
+      >
+        Submit
+      </button>
+    </div>
```
Comment (thecodechaser) [inline src/components/Addbook.js]: - [ ] Add book functionality is not working, Please make sure to pass values entered by the user in dispatch and on submit button, the added book should display on the screen.

File: src/components/Deletebook.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
+
+function Deletebook() {
+  const dispatch = useDispatch();
+  return (
+    <div>
+      <button type="button" onClick={() => dispatch(removeBook())}>Delete </button>
+    </div>
+  );
```
Comment (thecodechaser) [inline src/components/Deletebook.js]: - [ ] Remove book functionality is not working, Please make sure to pass the book id in dispatch, and on the delete button, the book should be removed from the screen.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @toyman640👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @toyman640, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




# Good Points 👍

- Descriptive Pull Request title & summary. ✔️
- `initialState` has been added to the `booksSlice` ✔️
- Has `<Provider>` component from `react-redux` ✔️
- Has `<BookList>` with `useSelector` ✔️
- Has a Component for Adding and Removing Books ✔️
- Good use of GitFlow ✔️
## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**